lib/maze/client/appium/bs_client.rb

```diff
@@ -67,7 +67,7 @@ def device_capabilities
 
         def log_run_intro
           # Log a link to the BrowserStack session search dashboard
-          url = "https://app-automate.browserstack.com/dashboard/v2/search?query=#{Maze.run_uuid}&type=builds"
+          url = "https://automate.browserstack.com/projects/#{project_name_capabilities[:project]}/builds/#{Maze.run_uuid}/1?tab=tests"
```

Comment:
Look like a copy paste error? app-automate has changed to automate.
```suggestion
          url = "https://app-automate.browserstack.com/projects/#{project_name_capabilities[:project]}/builds/#{Maze.run_uuid}/1?tab=tests"
```